Stripe/STPBankSelectionViewController.m

```diff
@@ -0,0 +1,136 @@
+}
+
+- (instancetype)initWithBankType:(STPBankType)bankType
+                   configuration:(STPPaymentConfiguration *)configuration
+                           theme:(STPTheme *)theme {
+    self = [super initWithTheme:theme];
+    if (self) {
+        NSCAssert(bankType == STPBankTypeFPX, @"STPBankSelectionViewController currently only supports FPX.");
+        _bankType = bankType;
+        _configuration = configuration;
+        _selectedBank = STPFPXBankBrandUnknown;
+        _apiClient = [[STPAPIClient alloc] initWithConfiguration:configuration];
+        self.title = STPLocalizedString(@"Bank Account", @"Title for bank account selector");
+    }
+    return self;
+}
+
+- (void)createAndSetupViews {
+    [super createAndSetupViews];
+
+    [self.tableView registerClass:[STPBankSelectionTableViewCell class] forCellReuseIdentifier:STPBankSelectionCellReuseIdentifier];
+
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+}
+
+- (void)updateAppearance {
+    [super updateAppearance];
+    
+    self.tableView.allowsSelection = YES;
+    
+    [self.tableView reloadData];
+}
+
+- (BOOL)useSystemBackButton {
+    return YES;
+}
+
+#pragma mark - UITableView
+
+- (NSInteger)numberOfSectionsInTableView:(__unused UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(__unused UITableView *)tableView numberOfRowsInSection:(__unused NSInteger)section {
+    return STPFPXBankBrandUnknown;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView
+         cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    STPBankSelectionTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:STPBankSelectionCellReuseIdentifier forIndexPath:indexPath];
+    STPFPXBankBrand bankBrand = indexPath.row;
+    BOOL selected = self.selectedBank == bankBrand;
+    [cell configureWithBank:bankBrand theme:self.theme selected:selected enabled:!self.loading];
+    return cell;
+}
+
+- (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cell forRowAtIndexPath:(NSIndexPath *)indexPath {
+    BOOL topRow = (indexPath.row == 0);
+    BOOL bottomRow = ([self tableView:tableView numberOfRowsInSection:indexPath.section] - 1 == indexPath.row);
+    [cell stp_setBorderColor:self.theme.tertiaryBackgroundColor];
+    [cell stp_setTopBorderHidden:!topRow];
+    [cell stp_setBottomBorderHidden:!bottomRow];
+    [cell stp_setFakeSeparatorColor:self.theme.quaternaryBackgroundColor];
+    [cell stp_setFakeSeparatorLeftInset:15.0f];
+}
+
+- (CGFloat)tableView:(__unused UITableView *)tableView heightForFooterInSection:(__unused NSInteger)section {
+    return 27.0f;
+}
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    if (self.loading) {
+        return; // Don't allow user interaction if we're currently setting up a payment method
```

Comment:
I think this might be better implemented using `- (BOOL)tableView:(UITableView *)tableView 
shouldHighlightRowAtIndexPath:(NSIndexPath *)indexPath`, that way you can't select at all -- or did you explicitly want to make the cells still interactable?

Reply:
The cells are enabled/disabled individually by the `userInteractionDisabled` flag set by `configureWithBank:theme:selected:enabled:`, which is copying the style of our other ViewControllers. They could probably use some rethinking — there's a lot of overreliance on [tableView reloadData] in our VCs.

Reply:
yikes okay, mind filing a JIRA for us to take a look at this?

Reply:
I might try to take care of it here, this PR isn't super urgent.